kura/org.eclipse.kura.linux.command.test/src/org/eclipse/kura/linux/command/CommandServiceTest.java

```diff
@@ -0,0 +1,54 @@
+/*******************************************************************************
+ * Copyright (c) 2016 Red Hat Inc and others
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Red Hat Inc
+ *******************************************************************************/
+
+package org.eclipse.kura.linux.command;
+
+import org.eclipse.kura.KuraException;
+import org.eclipse.kura.command.CommandService;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class CommandServiceTest {
+
+    private CommandService service;
+
+    @Before
+    public void setup() {
+        this.service = new CommandServiceImpl();
+    }
+
```

Comment:
Nothing is referenced here. Setting a null value doesn't really help in modern java implementations, so leaving this as is would be fine unless we target Java 1.0 again.